shoop/campaigns/models/basket_effects.py

```diff
@@ -0,0 +1,90 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from polymorphic.models import PolymorphicModel
+
+from shoop.admin.forms.fields import PercentageField
+from shoop.core.fields import MoneyValueField
+from shoop.utils.properties import PriceProperty
+
+
+class BasketEffect(PolymorphicModel):
+    identifier = None
+    model = None
+    admin_form_class = None
+    yields = False
+    name = _("Basket effect")
```

Comment:
I think the name Basket effect here is unnesseray since I think the name should be always defined by subclass.